src/Api/Vault/Models/Request/CipherRequestModel.cs

```diff
@@ -1,14 +1,14 @@
 ﻿using System.ComponentModel.DataAnnotations;
 using System.Text.Json;
-using Bit.Core.Entities;
-using Bit.Core.Enums;
-using Bit.Core.Models.Data;
 using Bit.Core.Utilities;
-using Core.Models.Data;
+using Bit.Core.Vault.Entities;
+using Bit.Core.Vault.Enums;
+using Bit.Core.Vault.Models.Data;
+using Core.Vault.Models.Data;
```

Comment:
Shouldn't this be `Bit.Core` instead of `Core`?

Reply:
Ah great catch, I accidentally left `Bit` off of the namespace of CipherDetails when I renamed it.